WPMedia/fusion-news-theme-blocks
# Themes 919 image height fix

## Description

- Promo images do not have have a set height/width and therefore are maintaining the aspect ratio of the original 
- This applies to both desktop and mobile 

## Jira Ticket

- [THEMES-919](https://arcpublishing.atlassian.net/browse/THEMES-919)

## Acceptance Criteria
https://arcpublishing.atlassian.net/browse/THEMES-919?focusedCommentId=859672

## Test Steps

1. Checkout this branch git checkout `THEMES-919-image-height-fix`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/results-list-block,@wpmedia/search-results-list-block`
3. Add block and make following changes

    website: Gazette
    content source: content-api-collections
    _id: CK2T4JQW5JFIFH4YDM5CNQAKPA


4. click update to see the results.

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-919]: https://arcpublishing.atlassian.net/browse/THEMES-919?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
